spec/observables/generate-spec.ts

```diff
@@ -2,36 +2,43 @@ import * as Rx from '../../dist/cjs/Rx.KitchenSink';
 import '../../dist/cjs/add/observable/generate';
 import {TestScheduler} from '../../dist/cjs/testing/TestScheduler';
 import {expect} from 'chai';
-declare const {expectObservable};
+declare const {asDiagram, expectObservable};
 declare const rxTestScheduler: TestScheduler;
 
 const Observable = Rx.Observable;
 
 describe('Observable.generate', () => {
-  it('should complete if condition does not meet', () => {
+  asDiagram('generate(1, x => false, x => x + 1)')
+  ('should complete if condition does not meet', () => {
     const source = Observable.generate(1, x => false, x => x + 1);
     const expected = '|';
 
     expectObservable(source).toBe(expected);
   });
-  it('should produce first value immediately', () => {
+
+  asDiagram('generate(1, x => x == 1, x => x + 1)')
+  ('should produce first value immediately', () => {
     const source = Observable.generate(1, x => x == 1, x => x + 1);
     const expected = '(1|)';
 
     expectObservable(source).toBe(expected, { '1': 1 });
   });
-  it('should produce all values synchronously', () => {
+
+  asDiagram('generate(1, x => x < 3, x => x + 1)')
```

Comment:
Usually we have only one of these `asDiagram` tests, per file. This third one is the most interesting for a diagram. Can you make the 1st and 2nd test revert back to normal `it()` test?